### ISSUE TITLE
Login screen uses different blue for background

### ISSUE THREAD

**Issue by rachelmcr** [Automattic/wp-desktop#676]
The login screen no longer shows the same blue as other Calypso screens.

Here's what it looks like in 4.3.0 (production release):

![image](https://user-images.githubusercontent.com/8658164/66665813-35259780-ec47-11e9-9778-b736a0ac9e9e.png)

Here's what it looks like now in 4.4.0-beta1:

![image](https://user-images.githubusercontent.com/8658164/66665816-3a82e200-ec47-11e9-82ee-1166846b3f34.png)

The only change is on that login screen — if you enter the signup flow in 4.4.0-beta1 you still see the Calypso blue:

<img width="1419" alt="Screenshot 2019-10-11 16 28 58" src="https://user-images.githubusercontent.com/8658164/66665780-23dc8b00-ec47-11e9-81e9-ffa9ce25dca0.png">

#### Context / Source
macOS 10.15 (Catalina)
#beta-testing

**Comment by rachelmcr** [comment]
It looks like Calypso screens use `color-wordpress-com` for the background color, but the login screen is using `color-primary` as set [here](https://github.com/Automattic/wp-calypso/blob/8e6f77c0c3f969d5e7cc418ad6d68662a0cf1d31/client/auth/style.scss#L30).

That line hasn't changed recently but @porada I see you've been making some updates to the semantic color naming in Calypso. Could you help with any updates/fixes that are needed to make sure the desktop app is using the correct colors?

**Comment by porada** [comment]
Thanks for the report!

This has already been flagged by the NUX folks. The default signup background should use `color-wordpress-com` (including [the line you highlighted](https://github.com/Automattic/wp-calypso/blob/8e6f77c0c3f969d5e7cc418ad6d68662a0cf1d31/client/auth/style.scss#L30)) for the visual continuity with our landing pages. Even though that continuity doesn’t really apply for our desktop app, using our brand color and maintaining the consistency stil feels likel a better idea.

With that said, looks like [this bit](https://github.com/Automattic/wp-calypso/blob/8e6f77c0c3f969d5e7cc418ad6d68662a0cf1d31/client/signup/style.scss#L2-L4) (that I think overwrites `color-primary` and is probably the reason behind this discrepancy) might be redundant.

**Comment by rachelmcr** [comment]
Thanks for taking a look!

> This has already been flagged by the NUX folks.

Is there a GitHub issue or other discussion I can follow to keep up with this? If this is fixed on the Calypso side we'll need to cherry pick that change into this desktop release.

**Comment by porada** [comment]
I don’t have anything other this comment [reporting a regression](https://github.com/Automattic/wp-calypso/pull/35949#issuecomment-527666835) during the bigger color update. I fixed that on the spot as `color-primary` feels oversaturated for full bleeds like on that page. It was designed with links and smaller interactive UI elements in mind.

**Comment by rachelmcr** [comment]
Thanks for the link! As I understand it, the issue that was spotted in that comment was with the signup flow, which you fixed — and that fix explains why the signup flow also uses the right color in the latest desktop app build.

Since the desktop login screen isn't used in Calypso web, I understand why it might not have been seen/fixed during that round of changes. Could you help us fix that color for the login screen used in the app?

**Comment by porada** [comment]
Sure, I can handle that. 🙂

Is there a way to access that app login screen in the browser?

**Comment by rachelmcr** [comment]
Hm, I'm not aware of any way to access that login screen in a browser; I think you'll need to run the app in dev mode ([details here](https://github.com/Automattic/wp-desktop/blob/develop/docs/development.md)) to see any changes you make to that screen.

Someone with more Calypso/desktop dev experience may have another suggestion, though - pinging @blowery in case you have advice on this.

**Comment by porada** [comment]
Yep, I’m already building the desktop app, but I’m not sure how to test this before deployment.

**Comment by mmtr** [comment]
It should be possible to visit `/oauth-login` on a local Calypso after enabling the `oauth` feature flag. I'll take a look and see if I can prepare a fix.
